compiler+runtime/include/cpp/jank/runtime/obj/inst.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <chrono>
```

Comment:
Not sure how to forward declare the types I use from this header, similar to how was done for [`uuid`](https://github.com/jank-lang/jank/blob/main/compiler%2Bruntime/include/cpp/jank/runtime/obj/uuid.hpp#L5-L8).

Reply:
I think we're already using `chrono` in our PCH. Don't worry about it for now.